portfolio/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```

Comment:
Please remove this TODO as the project ID has been set.

Reply:
Good catch! Fixed.